packages/vuetify/src/components/VLayout/VLayoutItem.tsx

```diff
@@ -1,34 +1,46 @@
+// Styles
+import './VLayoutItem.sass'
+
 // Composables
 import { makeLayoutItemProps, useLayoutItem } from '@/composables/layout'
 
 // Utilities
-import { computed, defineComponent, toRef } from 'vue'
+import { defineComponent, toRef } from 'vue'
 import { makeProps } from '@/util'
 
 // Types
-import type { Prop } from 'vue'
+import type { PropType } from 'vue'
 
 export default defineComponent({
   name: 'VLayoutItem',
 
   props: makeProps({
     position: {
-      type: String,
+      type: String as PropType<'top' | 'right' | 'bottom' | 'left'>,
       required: true,
-    } as Prop<'top' | 'right' | 'bottom' | 'left'>,
+    },
+    size: {
+      type: [Number, String],
+      default: 300,
```

Comment:
Should we assign a default here or make it required?

Reply:
I think a default value works